lib/src/widgets/chat_view.dart

```diff
@@ -211,83 +211,97 @@ class _ChatViewState extends State<ChatView>
+                      )
+                    else if (chatViewState.isError)
+                      ChatViewStateWidget(
+                        chatViewStateWidgetConfig:
+                            chatViewStateConfig?.errorWidgetConfig,
+                        fallbackTitle: "Error",
+                        chatViewState: chatViewState,
+                        onReloadButtonTap:
+                            chatViewStateConfig?.onReloadButtonTap,
+                      )
+                    else if (chatViewState.hasMessages)
+                      Stack(
+                        clipBehavior: Clip.none,
+                        children: [
+                          Positioned.fill(
+                            bottom: -MediaQuery.of(context).size.height,
+                            child: ValueListenableBuilder<ReplyMessage>(
+                              valueListenable: replyMessage,
+                              builder: (_, state, child) {
+                                return ChatListWidget(
+                                  replyMessage: state,
+                                  chatController: widget.chatController,
+                                  chatBackgroundConfig:
+                                      widget.chatBackgroundConfig,
+                                  reactionPopupConfig:
+                                      widget.reactionPopupConfig,
+                                  typeIndicatorConfig:
+                                      widget.typeIndicatorConfig,
+                                  chatBubbleConfig: widget.chatBubbleConfig,
+                                  loadMoreData: widget.loadMoreData,
+                                  isLastPage: widget.isLastPage,
+                                  replyPopupConfig: widget.replyPopupConfig,
+                                  loadingWidget: widget.loadingWidget,
+                                  messageConfig: widget.messageConfig,
+                                  profileCircleConfig:
+                                      widget.profileCircleConfig,
+                                  repliedMessageConfig:
+                                      widget.repliedMessageConfig,
+                                  swipeToReplyConfig: widget.swipeToReplyConfig,
+                                  onChatListTap: widget.onChatListTap,
+                                  assignReplyMessage: (message) =>
+                                      _sendMessageKey.currentState
+                                          ?.assignReplyMessage(message),
+                                  emojiPickerSheetConfig:
+                                      widget.emojiPickerSheetConfig,
+                                );
+                              },
+                            ),
+                          )
+                        ],
+                      )
+                    else
+                      Container(),
+                  ][0],
```

Comment:
Can you please avoid this? Creating a list with one element and then accessing it?